### PR TITLE
Add article about specification of requests from #283

### PR DESCRIPTION
**Pull request by zner0L**

### CODE REVIEW

File: content/de/blog/request-specification/index.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+{
+    "title": "Präzisierung einer Auskunftsanfrage gemäß Erwägungsgrund 63 DSGVO",
+    "slug": "praezisierung-einer-auskunftsanfrage",
+    "date": "2020-05-18T16:20:00+02:00",
+    "type": "blog",
+    "description": "Gemäß Erwägungsgrund 63 DSGVO können Verantwortliche eine Präziesierung von Auskunftsanfragen verlagen. Aber wann ist das gerechtfertig? Und wie muss eine solche Präszisierung aussehen?",
```
Comment (baltpeter) [inline content/de/blog/request-specification/index.md:6]:
```suggestion
    "description": "Gemäß Erwägungsgrund 63 DSGVO können Verantwortliche eine Präzisierung von Auskunftsanfragen verlangen. Aber wann ist das gerechtfertigt? Und wie muss eine solche Präzisierung aussehen?",
```

File: content/de/blog/request-specification/index.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    "tags": [ "dsgvo", "Erwägungsgrund 63", "präzisieren", "präzisierung", "konkretisieren", "konkretisierung", "artikel 15", "Auskunftsantrag", "gdpr" ],
+    "authors": [ "andreas_bampounis" ],
+    "featured_image": "praezisierung-von-auskunftsanfragen",
+    "license": "cc-by-4.0",
+    "notices": "Titelfoto angepasst nach: „[The files](https://unsplash.com/photos/Q9y3LRuuxmg)“ von [Maksym Kaharlytskyi](https://unsplash.com/@qwitka) ([Unsplash-Lizenz](https://unsplash.com/license))"
+}
+
+Gemäß Art. 15 DSGVO und konkreter dem Erwägungsgrund 63 S. 7[^1] darf der_die Verantwortliche der betroffenen Person eine Präzisierung im Rahmen ihrer Ausübung des Auskunftsrechts verlangen. Um diese Aufforderung verlangen zu können, muss der_die Verantwortliche eine große Menge von Daten in Bezug auf die betroffene Person verarbeiten.
```
Comment (baltpeter) [inline content/de/blog/request-specification/index.md:14]:
```suggestion
Gemäß Art. 15 DSGVO und konkreter dem Erwägungsgrund 63 S. 7[^1] darf der_die Verantwortliche von der betroffenen Person eine Präzisierung der Anfrage im Rahmen ihrer Ausübung des Auskunftsrechts verlangen. Um diese Aufforderung verlangen zu können, muss der_die Verantwortliche eine große Menge von Daten in Bezug auf die betroffene Person verarbeiten. 
```

File: content/de/blog/request-specification/index.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+## Präzisierung
+
+Neben der juristischen Auslegung der Normen ist für die Praxis besonders wichtig, wie die betroffene Person ihre Auskunftsanfrage präzisieren soll, wenn dies vom Verantwortlichen verlangt wird.
+
+Am Anfang sollte die Anfragesteller_in klarstellen, an welchen Informationen und Verarbeitungsvorgängen er_sie interessiert ist.[^8] Eine bestimmte Kategorie von Daten i.S.v. Art. 15 DSGVO sind üblicherweise Leistungs- und Verhaltensdaten.[^9] Die Anfrage ist bestimmt genug, wenn die betroffene Person eine Auskunft der verarbeiteten und nicht der sich in ihrer Personalakte befindenden Daten begehrt. Nicht in die Anfrage aufgenommen werden können oder müssen die Daten, die Anfragesteller_in nicht kennen kann.[^10] Eine Präzisierung ist auch bestimmt genug, wenn die Datenkategorie und der Zeitraum festgelegt worden sind (z.B. E-Mails von einem bestimmten Zeitraum).[^11] Letztere Methode ist zu empfehlen.
```
Comment (baltpeter) [inline content/de/blog/request-specification/index.md:42]:
```suggestion
Am Anfang sollte die Anfragesteller_in klarstellen, an welchen Informationen und Verarbeitungsvorgängen er_sie interessiert ist.[^8] Eine bestimmte Kategorie von Daten i.S.v. Art. 15 DSGVO sind üblicherweise Leistungs- und Verhaltensdaten.[^9] Die Anfrage ist bestimmt genug, wenn die betroffene Person eine Auskunft der verarbeiteten und nicht der sich in ihrer Personalakte befindenden Daten begehrt. Nicht in die Anfrage aufgenommen werden können oder müssen die Daten, welche die Anfragesteller_in nicht kennen kann.[^10] Eine Präzisierung ist auch bestimmt genug, wenn die Datenkategorie und der Zeitraum festgelegt worden sind (z.B. E-Mails von einem bestimmten Zeitraum).[^11] Letztere Methode ist zu empfehlen.
```

**Comment by zner0L** [comment]
Rebased and changes applied.

**Pull request merged**